mysql-test/suite/sys_vars/t/innodb_encrypt_tables_basic.test

```diff
@@ -0,0 +1,27 @@
+--source include/have_innodb.inc
+
+SELECT @@GLOBAL.innodb_encrypt_tables;
+
+--error ER_GLOBAL_VARIABLE
+SET SESSION innodb_encrypt_tables="ON";
+
+SET GLOBAL innodb_encrypt_tables="ON";
+SELECT @@GLOBAL.innodb_encrypt_tables;
+SET GLOBAL innodb_encrypt_tables="OFF";
+SELECT @@GLOBAL.innodb_encrypt_tables;
+SET GLOBAL innodb_encrypt_tables="FORCE";
+SELECT @@GLOBAL.innodb_encrypt_tables;
+
+# Incorrect values
+--error ER_WRONG_TYPE_FOR_VAR
+SET GLOBAL innodb_encrypt_tables=1.1;
+--error ER_WRONG_TYPE_FOR_VAR
+SET GLOBAL innodb_encrypt_tables=1e1;
+--error ER_WRONG_VALUE_FOR_VAR
+SET GLOBAL innodb_encrypt_tables='foo';
+
+SELECT @@GLOBAL.innodb_encrypt_tables;
+
+# If we are lacking permissions, then we cannot restore the startup value
+# at the end.
+--source include/restart_mysqld.inc
```

Comment:
? If we are lacking permissions, then we could not switch away from the startup value neither. And how can the MTR MySQL user lack permissions?

Reply:
I based this on another testcase, and this was included in it - I assumed it made sense. I'll remove it then.